metron-interface/metron-alerts/cypress/support/commands.js

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
This whole file appears to be commented out. Is this needed?

Reply:
Some files look like placeholders but cypress needs them. Cypress regenerates all at a run if we remove them. I think it's easier to keep them in Git instead of cleaning them every time we running tests locally.